package/MDAnalysis/analysis/align.py

```diff
@@ -630,7 +630,8 @@ def rms_fit_trj(
         tol_mass=0.1,
         strict=False,
         force=True,
-        quiet=False,
+        verbose=None,
+        quiet=None,
```

Comment:
leave verbose in for now. But I moving it to the end of the argument list and raising a warning should help users understand that this is a transition period to get used to the new keyword.